scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -158,8 +157,8 @@ coldata_df <- colData(integrated_sce) |>
   as.data.frame() |>
   # merge with sample metadata 
   dplyr::left_join(sample_metadata, by = c("sample" = "library_id")) |>
-  # create a new columns
-  # combination of barcode and sample
+  # create new columns
+  # cell_id is combination of barcode and sample
```

Comment:
```suggestion
  # cell_id is the combination of barcode and sample
```

Reply:
Noting I've just seen that @jashapiro also left a review here but with "a" instead of "the," and either is fine!